apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -272,12 +291,13 @@ NetSimRouterLogModal.prototype.newRender_ = function () {
       i18n={i18n}
       canSetRouterLogMode={this.canSetRouterLogMode_()}
       isAllRouterLogMode={this.isAllRouterLogMode_}
-      setRouterLogMode={this.setRouterLogMode_.bind(this)}
+      setRouterLogMode={this.boundSetRouterLogMode_}
       localAddress={this.localNode_ ? this.localNode_.getAddress() : undefined}
       currentTrafficFilter={this.currentTrafficFilter_}
-      setTrafficFilter={this.setTrafficFilterMode_.bind(this)}
+      setTrafficFilter={this.boundSetTrafficFilterMode_}
```

Comment:
It's considered bad form to bind functions inside the render method because you'll re-bind the function on every render call (when we really only need to do it once).  Hence this change to keep bound callback methods.